### DISCUSSION TITLE
PCL安装整合包报错

### DISCUSSION THREAD

**Discussion by ChemEng227** [Meloong-Git/PCL#1585] [提问] [answered]
试图安装的整合包为GTNH 2.3.2 Java17-20版本
报错信息如下：
整合包安装失败
→ 解压整合包文件失败
→ 指定的路径或文件名太长，或者两者都太长。完全限定文件名必须少于 260 个字符，并且目录名必须少于 248 个字符。
   在 PCL.ModModpack.UnpackFiles(String InstallTemp, String FileAddress)
   在 PCL.ModModpack._Closure$__9-0._Lambda$__0(LoaderTask`2 Task)
   在 PCL.ModLoader.LoaderTask`2._Closure$__12-0._Lambda$__0()
错误类型：System.IO.PathTooLongException

**Comment by MoYuan-CN** [comment]
路径太长 :(
请自行修改对应安装路径并使其短于要求

**Comment by WForst-Breeze** [comment, marked as answer]
请见 #1448，如果存在此问题，请向整合包作者进行反馈或尝试缩短安装路径
